spec/requests/api/users_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-describe "Users API" do
+context "Users API" do
   describe "GET/users/:id" do
```

Comment:
I notice we're using `context` and `describe` interchangeably here. Why is that?